src/components/modals/EditUserModal.js

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import Modal from "@material-ui/core/Modal";
+import Backdrop from "@material-ui/core/Backdrop";
+import Fade from "@material-ui/core/Fade";
+import EditIcon from "@material-ui/icons/Edit";
+// import Input from "../input/input";
+// import Button from "../input/Button";
+import AddIcon from "@material-ui/icons/Add";
+import { useFormik } from "formik";
+import "../../App.css";
+import { useStyles } from "./styles";
+import TextField from "@material-ui/core/TextField";
+import { addUsers, editUsers } from "../../services/userApi"
+
+function EditUserModal({ row, name }) {
+  const classes = useStyles();
+  const [open, setOpen] = React.useState(false);
+
+  const handleOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const formik = useFormik({
+    initialValues: {
+      email: row?.email ? row.email : "",
+      first_name: row?.first_name ? row.first_name : "",
+      last_name: row?.last_name ? row.last_name : "",
```

Comment:
you can rewrite it as below:
`first_name: row?.first_name  || ""`

Reply:
done